.travis.sh

```diff
@@ -12,7 +12,12 @@ set -e # Error build immediately if install script exits with non-zero
 # if this is a docker job, run in the container instead; but if not just run it here.
 if [ -n "$DOCKER_IMAGE" ]; then
     ./util/dockerize.sh ./.ci.sh
+elif [ "${TRAVIS_OS_NAME}" = "windows" ]; then
+    choco install wsl-ubuntu-1804
+    brun="/c/ProgramData/chocolatey/lib/wsl-ubuntu-1804/tools/unzipped/ubuntu1804.exe run"
+    $brun sudo apt update
+    $brun sh -c 'DEBIAN_FRONTEND="noninteractive" sudo apt install -y gcc git curl'
```

Comment:
Because of

> WARNING: apt does not have a stable CLI interface. Use with caution in scripts.

```suggestion
    $brun sudo apt-get update
    $brun sh -c 'DEBIAN_FRONTEND="noninteractive" sudo apt-get install -y gcc git curl'
```